.github/workflows/ci.release.yml

```
@@ -0,0 +1,41 @@
+name: CI - Release
+
+on:
+  pull_request:
+    branches: [ release ]
+  workflow_dispatch:
+
+jobs:
+  dependency-check:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        directory: ['./packages/backend', './packages/frontend']
+    steps:
+    - name: "Checkout repository"
+      uses: actions/checkout@v2
+
+    - name: "Set up Node.js 12.x"
+      uses: actions/setup-node@v2.4.1
+      with:
+        node-version: 12.x
```

```suggestion
    - name: "Set up Node.js 16.x"
      uses: actions/setup-node@v2.4.1
      with:
        node-version: 16.x
```
Upgrading to a newer node version might be considered

---

Right, but not tested yet :sweat: There is also an uncaught promise, which needs to be located and resolved first as this can cause the application to fail :grin:

---

Yeah, who cares, unhandled promise rejections were deprecated anyways :sunglasses: Just kidding, do you have any hint where the promise might be located?